File: Source/Back-end/src/main/java/org/elis/manutenzione/dto/response/AggiungiLuogoResponse.java
package org.elis.manutenzione.dto.response;

import lombok.Data;

/**
 * DTO che rappresenta la risposta alla richiesta di aggiunta di un luogo.
 */
@Data
public class AggiungiLuogoResponse {

    /**
     * L'ID del luogo aggiunto.
     */
    private long id;

    /**
     * Il nome del luogo.
     */
    private String nome;

    /**
     * Il tipo di luogo (es. Bagno, Stanza, Luogo Comune).
     */
    private String tipo;

    /**
     * Il nucleo a cui appartiene il luogo.
     */
    private String nucleo;

    /**
     * La capienza del luogo.
     */
    private int capienza;
}

File: Source/Front-end/front-end/src/app/model/response/aggiungi-luogo-response.ts
export class AggiungiLuogoResponse{
  id: number;
  nome: string;
  tipo: string;
  nucleo: string;
  capienza: number;

  constructor(id: number, nome: string, tipo: string, nucleo: string, capienza: number) {
    this.id = id;
    this.nome = nome;
    this.tipo = tipo;
    this.nucleo = nucleo;
    this.capienza = capienza;
  }
}
